### ISSUE TITLE
test/example.js fail travis ci

### ISSUE THREAD

**Issue by mj3cheun** [maxkfranz/slush-js#4]
By default, the line to require chai is as follows:

```
let { expect, assert } = require('chai');
```

This fails travis ci tests for node versions 5 and below because it doesn't like the curl brackets and let.

I found that changing the code to the following fixed the problem:

```
var expect = require('chai').expect;
var assert = require('assert');
```

**Comment by maxkfranz** [comment]
@mj3cheun This repo wasn't in my feed so I missed this.  I'll take a look

**Comment by maxkfranz** [comment]
Published as 1.1.4
